src/data-capsule.js

```diff
@@ -22,7 +25,7 @@ export default class DataCapsule extends BaseStorage {
     return options;
   }
 
-  setItem(key, value, options) {
+  async setItem(key, value, options) {
```

Comment:
Why is this needed? 🤔

Reply:
Because `setItem` is an async operation, that's the contract. Before that change it was very bad, it could `throw` out of the promise chain, and **just then** it returned a promise. Note that as a part of the commit I removed the `return Promise.resolve()` line (from local storage strategy) and added an `async` annotation instead.

Reply:
Yes, I didn't fully realize until now that `async` annotation causes a function to return a promise even if it didn't include an explicit `return` statement.

Thanks!